### ISSUE TITLE
segmentation fault rendering tables with \Sundown\Markdown

### ISSUE THREAD

**Issue by lorderb** [chobie/php-sundown#26]
Example:

```
<?php
$table = <<<EOS
| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |
EOS;

//works:
$sd = new Sundown($table, array('tables'=>true));
echo $sd->toHTML();

//sementation fault:
$md = new \Sundown\Markdown(\Sundown\Render\HTML, array('tables'=>true)); 
echo $md->render($table);
```

Versions:
Sundown Support => enabled
Version => 0.3.6
Sundown Version => 1.15.2

**Comment by chobie** [comment]
i'll upload fix release to PECL tomorrow. (now, AM3 in Japan.)
thanks reporting an issue.

**Comment by chobie** [comment]
released 0.3.7 :star:

**Comment by hooblei** [comment]
that was quick _g_ - thanks a lot
